core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -183,7 +183,7 @@
   </div>
 
   <ul>
-    <li [ngClass]="{'active': currentUrl === '/learn/math'}">
+    <li [ngClass]="{'active': currentUrl === '/learn{{DEFAULT_CLASSROOM_URL_FRAGMENT}}'}">
```

Comment:
```suggestion
    <li [ngClass]="{'active': currentUrl === '/learn/{{DEFAULT_CLASSROOM_URL_FRAGMENT}}'}">
```
please fix it to all other places also!

Reply:
Done!